src/bp/ui-shared/src/Toaster/index.tsx

```diff
@@ -3,36 +3,56 @@ import _ from 'lodash'
 
 import { lang } from '../translations'
 
-export enum Timeout {
-  SHORT = 1000,
-  MEDIUM = 3000,
-  LONG = 5000
-}
-
 export interface ToastOptions {
-  delayed: boolean
-  timeout: Timeout
+  // Delaying to avoid the react lifecycle issue (executing before it is rendered)
+  delayed?: boolean
+  timeout?: 'short' | 'medium' | 'long'
   onDismiss?: (didTimeoutExpire: boolean) => void
 }
 
-export const toastFailure = (
+const success = (
```

Comment:
It would be great if we added some jsdoc comments on everything in shared UI. This way no need to check the implementation when using it